templates/local.conf.j2

```diff
@@ -0,0 +1,16 @@
+[Time]
+{% if ntp_servers is defined %}
+NTP={{ ntp_servers | join(" ") }}
+{% else %}
+#NTP=
+{% endif %}
+{% if ntp_fallback_servers is defined %}
+FallbackNTP={{ ntp_fallback_servers | join(" ") }}
+{% else %}
+#FallbackNTP=0.arch.pool.ntp.org 1.arch.pool.ntp.org 2.arch.pool.ntp.org 3.arch.pool.ntp.org
+{% endif %}
+#RootDistanceMaxSec=5
+#PollIntervalMinSec=32
```

Comment:
Same here, this is drop-in, so we can be as minimal as possible, please remove all the commented out lines that don't do anything.